metaflow/cli.py

```diff
@@ -649,14 +649,32 @@ def run(obj,
               default=DEFAULT_RUN_NAME,
               help="name assigned to the new KFP run"
               )
+@click.option('--namespace',
+              'namespace',
+              default=None,
+              help="namespace of your run in KFP."
+              )
+@click.option('--userid',
+              'userid',
+              default=None,
+              help="your user ID (your ZG email)."
+              )
 @click.pass_obj
 def run_on_kfp(obj,
         code_url=DEFAULT_FLOW_CODE_URL,
         experiment_name=DEFAULT_EXPERIMENT_NAME,
-        run_name=DEFAULT_RUN_NAME
+        run_name=DEFAULT_RUN_NAME,
+        namespace=None,
+        userid=None
         ):
+    
+    namespace_arg = namespace if namespace is not None else KFP_SDK_NAMESPACE
```

Comment:
Why not set KFP_SDK_NAMESPACE the default value for the parameter? That is what this line is doing.

Reply:
Good catch, will do that now.

Reply:
Resolved.